openedx/core/djangoapps/api_admin/migrations/0002_auto_20160325_1604.py

```diff
@@ -1,36 +0,0 @@
-# -*- coding: utf-8 -*-
```

Comment:
What's going on with this migration? We cannot simply delete migrations this way.